### PR TITLE
Feature: introduce RuboCop performance

### PR DESCRIPTION
**Pull request by tagliala**
Hi,

I'm proposing to add RuboCop performance to avoid simple mistakes that can improve performance, like capturing matchdata when checking regexp instead of using `match?`

### CODE REVIEW

**Comment by sds** [comment]
Generally supportive. Can we get the rest of the suite passing? Thank you.

File: Appraisals [hunk 1]
```diff
@@ -32,5 +32,5 @@ appraise 'haml6.3' do
 end
 
 appraise 'rubocop1.0' do
-  gem 'rubocop', '~> 1.0.0'
+  gem 'rubocop', '~> 1.0'
```
Comment (tagliala) [inline Appraisals:35]: all green, but I had to change this

**Comment by tagliala** [comment]
Probably it would be better to drop support for old RuboCop versions before

**Comment by sds** [comment]
Thanks!

**Pull request merged**